t/integration/test_canvas.py

```diff
@@ -701,6 +703,18 @@ def test_nested_group_group(self, manager):
         res = sig.delay()
         assert res.get(timeout=TIMEOUT) == [42, 42]
 
+    @pytest.mark.xfail(raises=DecodeError, strict=True)
```

Comment:
Should this still fail or does it fail because our test suite doesn't switch the configuration because we use the session worker?